src/request/dispatchRequest.ts

```diff
@@ -0,0 +1,31 @@
+import TestRun from '../test-run';
```

Comment:
The `dispatch` word is usually used for events. Rename this file to `src/request/send`.

Remove the `request` word from all related file names. Class names in it should not be changed.